ResearchKit/Common/ORKAnswerFormat_Internal.h

```diff
@@ -1,5 +1,6 @@
 /*
  Copyright (c) 2015, Apple Inc. All rights reserved.
+ Copyright (c) 2015, Scott Guelich.
```

Comment:
Clearly nothing. :smile: Xcode's class counterparts tracking sometimes gets confused in the latest version, and I probably edited this thinking it was ORKAnswerFormat.h.